lib/pages/App.js

```diff
@@ -173,12 +203,41 @@ export default class App extends Component {
             targetId={uploadTargetUser}
             timezoneIsSelected={Boolean(this.props.selectedTimezone)}
             userDropdownShowing={this.props.showingUserSelectionDropdown}
+            isClinicAccount={this.props.isClinicAccount}
             userIsSelected={uploadTargetUser !== null}
             addDevice={this.props.sync.addTargetDevice}
             removeDevice={this.props.sync.removeTargetDevice}
             onDone={this.props.async.clickDeviceSelectionDone} />
         </div>
       );
+    } else if (page === pages.CLINIC_USER_SELECT) {
+      const { allUsers, page, targetUsersForUpload, uploadTargetUser } = this.props;
+      return (
+        <div>
+          <ClinicUserSelect
+            allUsers={allUsers}
+            onUserChange={this.props.async.setUploadTargetUserAndMaybeRedirect}
+            page={page}
+            targetId={uploadTargetUser}
+            targetUsersForUpload={targetUsersForUpload}
+            onAddUserClick={this.props.async.clickAddNewUser}
+            setTargetUser={this.props.sync.setUploadTargetUser} />
+        </div>
+      );
+    } else if (page === pages.CLINIC_USER_EDIT) {
+      const { allUsers, page, targetUsersForUpload, uploadTargetUser } = this.props;
+      return (
+        <div>
+          <ClinicUserEdit
+            targetId={uploadTargetUser}
+            allUsers={allUsers}
+            loggedInUser={this.props.loggedInUser}
+            createUser={this.props.async.createCustodialAccount}
+            updateUser={this.props.async.clickEditUserNext}
+            cancelEdit={this.handleClickChangePerson.bind(this)}
```

Comment:
Move the binding into the constructor like we do for `handleClickChooseDevices` and `handleDismissDropdown`? For this and for `handleClickEditUser`, plz.